src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,60 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int FIGURE_COUNT = 5;
+    public static final int MAX_NUMBER = 28;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+    private String randomColor = colorSupplier.getRandomColor();
```

Comment:
let's use `String randomColor = colorSupplier.getRandomColor();` in methods that return random figures